### PR TITLE
Add port parameter

### PR DESCRIPTION
**Pull request by KamilKurde**
## The problem
Currently, this action uses ADB commands without specifying device/emulator to use.
This is fine for GitHub-hosted machines, as they are isolated from each other, but becomes a problem when using this action on self-hosted machine, as ADB commands will fail if there are multiple devices present, and it wasn't specified which one to use.

## The fix
Adding `emulator-port` input, that allows a job to specify port to start emulator on.
This way, one machine could have up to 16 emulators running at the same time (due to constraints in port range) without interfering with each other.

### CODE REVIEW

File: __tests__/input-validator.test.ts [hunk 1]
```diff
@@ -172,6 +173,33 @@ describe('force-avd-creation validator tests', () => {
   });
 });
 
+describe('emulator-port validator tests', () => {
+  it('Validates if emulator-port is even and in range', () => {
+    const func = () => {
+      validator.checkPort(5554);
+    };
+    expect(func).not.toThrow();
+  });
```
Comment (AKJAW) [inline __tests__/input-validator.test.ts:182]: Might be good to also test that the MAX_PORT value also is valid, since it's the other boundary value

Comment (KamilKurde) [reply __tests__/input-validator.test.ts:182]: Isn't this already tested two tests later?

Comment (AKJAW) [reply __tests__/input-validator.test.ts:182]: Somewhat yes, but it uses the MAX_PORT variable directly, not its value, so changing MAX_PORT value would always make the test pass, however from the reqs we know that 5584 is the max and it will never change. Having such test and editing the MAX_PORT value would result in failure which is what we want

**Comment by pankajkakkar** [comment]
May I please add a vote for getting this PR merged? It doesn't sound like there are any blockers from the conversation above. I'm facing this issue as well, and every time I want to run a workflow on my repo I have to shut down any running emulators on my Mac.

There's also an open issue https://github.com/ReactiveCircus/android-emulator-runner/issues/293 on this topic.
